src/components/event/update/EventUpdateForm.vue

```diff
@@ -0,0 +1,201 @@
+                    </v-col>
+                </v-row>
+                <h3>{{ $t("tookPlaceLabel") }}</h3>
+                <v-row>
+                    <v-col v-if="timePeriodInput" cols="3">
+                        <date-picker
+                            v-model="dateFrom"
+                            :label="$t('fromLabel') + '*'"
+                            color="primary"
+                        ></date-picker>
+                    </v-col>
+                    <v-col v-if="timePeriodInput" cols="3">
+                        <date-picker
+                            v-model="dateTo"
+                            :label="$t('toLabel') + '*'"
+                            color="primary"
+                        ></date-picker>
+                    </v-col>
+                    <v-col v-if="!timePeriodInput" cols="6">
+                        <v-text-field
+                            v-model="eventYear" type="number" :min="1950" :max="2030"
+                            :label="$t('eventYearLabel') + '*'" :rules="requiredFieldRules"></v-text-field>
+                    </v-col>
+                    <v-col>
+                        <v-btn color="blue darken-1" compact @click="timePeriodInput = !timePeriodInput">
+                            {{ timePeriodInput ? $t("dontKnowExactDateLabel") : $t("knowExactDateLabel") }}
+                        </v-btn>
+                    </v-col>
+                </v-row>
+                <v-row>
+                    <v-col cols="10">
+                        <v-autocomplete v-model="state" :label="$t('stateLabel')" :items="countryList" return-object></v-autocomplete>
+                    </v-col>
+                </v-row>
+                <v-row>
+                    <v-col>
+                        <multilingual-text-input ref="placeRef" v-model="place" :label="$t('placeLabel')" :initial-value="toMultilingualTextInput(presetEvent?.place, languageTags)"></multilingual-text-input>
+                    </v-col>
+                </v-row>
+                <v-row>
+                    <v-col cols="5">
+                        <v-text-field
+                            v-model="conferenceNumber"
+                            :label="$t('conferenceNumberLabel')"
+                        ></v-text-field>
+                    </v-col>
+                    <v-col cols="5">
+                        <v-text-field
+                            v-model="entryFee"
+                            :label="$t('entryFeeLabel')"
+                        ></v-text-field>
+                    </v-col>
+                </v-row>
+                <v-row>
+                    <v-checkbox v-model="serialEvent" :label="$t('serialEventLabel')"></v-checkbox>
+                </v-row>
+            </v-col>
+        </v-row>
+
+        <v-row>
+            <p style="margin-left: 20px;">
```

Comment:
might be problematic later for improvements of design, better to move to css file

Reply:
You are right, I will refactor the codebase and replace these occurences of style for required fields message